config/options.go

```diff
@@ -237,6 +245,7 @@ var defaultOptions = Options{
 	GRPCServerMaxConnectionAgeGrace: 5 * time.Minute,
 	CacheStore:                      "autocache",
 	AuthenticateCallbackPath:        "/oauth2/callback",
+	CertFolder:                      filepath.Join(".", ".pomerium"),
```

Comment:
❓ Should we have a default value for cert folder?

Reply:
`.` might not be very predictable.

If we provide a default, it should be relative to `/etc`, `$HOME` or from https://github.com/adrg/xdg like `XDG_CACHE_HOME`.  The latter is nice in that it is cross platform and idiomatic to each platform.

Reply:
@travisgroth good idea. What do you think about https://github.com/pomerium/pomerium/pull/644/commits/b391d01326b5aace6e47d45239f09315ddfed412

Reply:
Given this:

>By default, CertMagic stores assets on the local file system in $HOME/.local/share/certmagic (and honors $XDG_DATA_HOME if set). CertMagic will create the directory if it does not exist. If writes are denied, things will not be happy, so make sure CertMagic can write to it!

Proposal: Let's use their default (which seems rational - and I think is approximately what https://github.com/pomerium/pomerium/commit/b391d01326b5aace6e47d45239f09315ddfed412 does?) and update our Docker images to set `$XDG_DATA_HOME` to `/data/`.  This leaves a nice `docker run -it -v pomerium:/data/pomerium ... pomerium/pomerium` pattern for container execution with a volume for storing certs. 

Thoughts?

Reply:
Good thinking. It also means we can do away with the explicit certs directory.

Reply:
Clarity - use the approximation of their pattern so that we store in `$XDG_DATA_HOME/pomerium`.